third_party/2and3/flask/testing.pyi

```diff
@@ -6,17 +6,7 @@ from typing import IO, Any, Iterable, Mapping, Optional, Text, TypeVar, Union
 
 from click import BaseCommand
 from click.testing import CliRunner, Result
-from werkzeug.test import Client
-
-def make_test_environ_builder(
-    app: Any,
-    path: str = ...,
-    base_url: Optional[Any] = ...,
-    subdomain: Optional[Any] = ...,
-    url_scheme: Optional[Any] = ...,
-    *args: Any,
-    **kwargs: Any,
-): ...
```

Comment:
While `make_test_environ_builder()` has been deprecated in the last released version, it still exists, so we should keep it for now.